applications/nrf_desktop/README.rst

```diff
@@ -331,6 +331,44 @@ Add the ``pm_static_${CMAKE_BUILD_TYPE}.yml`` partition manager configuration fi
 
 For more information about how to configure the flash memory layout using the partition manager, see :ref:`partition_manager`.
 
+Device Firmware Upgrade
+***********************
+
+The nRF Desktop application uses background image transfer for the DFU process.
+The following elements take part in the firmware update process:
+
+    * `Update image generation`_
+    * `Update image transfer`_
+    * `Update image verification and swap`_
+
+Update image generation
+=======================
+
+The update image is automatically generated if the bootloader is enabled and can be found at the application build directory.
+
+Update image transfer
+=====================
+
+The update image is transmitted through the :ref:`nrf_desktop_config_channel` while the device is in normal operation.
+The device must be connected to the host via the USB or the Bluetooth.
+The update image data is transmitted using HID feature reports.
+
+At the application side the process is handled by :ref:`nrf_desktop_dfu`.
+Please refer to the module documentation for how it should be enabled and configured.
+
+At the host side the process is handled by the :ref:`nrf_desktop_config_channel_script`.
+Please refer to the tool documentation for more information how execute the DFU process on the host.
+
+Update image verification and swap
+==================================
+
+Once the update image transfer is completed, the DFU process will continue after the device reboot.
```

Comment:
Should I trigger the reboot? Because your documentation suggests it is happening automatically.

Reply:
Changed this text a bit.
When you use our hid configurator the reboot will be done automatically. I am not sure any client will actually use it but we don't really know.